rag-sql-agent/app.py

```
@@ -0,0 +1,332 @@
+            storage_context = StorageContext.from_defaults(vector_store=vector_store)
+            return VectorStoreIndex.from_documents(
+                documents,
+                storage_context=storage_context,
+                show_progress=True
+            )
+
+@st.cache_resource
+def initialize_workflow(_index, _sql_database):
+    with st.spinner("🚀 Starting query engines..."):
+        # SQL Query Engine
+        sql_context = """The table 'city_stats' contains:
```

Ruff (0.8.2)

Local variable `sql_context` is assigned to but never used

Remove assignment to unused variable `sql_context`

(F841)